sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -25,6 +25,23 @@ public class CommunicationIdentityClientLiveTests : CommunicationIdentityClientL
         /// Options used to exchange an AAD access token of a Teams user for a new Communication Identity access token.
         /// </summary>
         private GetTokenForTeamsUserOptions CTEOptions = new GetTokenForTeamsUserOptions("Sanitized", "Sanitized", "Sanitized");
+
+        private const string TOKEN_EXPIRATION_OVERFLOW_MESSAGE = "The tokenExpiresIn argument is out of permitted bounds [1,24] hours. Please refer to the documentation and set the value accordingly.";
+        private const double ALLOWED_DEVIATION = 0.05;
```

Comment:
allowed deviation for what? :) It’s better to have proper names for constant to understand immediately for what purpose it was created or for which features

Reply:
Renamed :-)